modules/serverless-admin-monitoring-eventing-channel.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-admin-monitoring-eventing-channel_{context}"]
+= Monitoring Knative Eventing channels
+
+You can use the {product-title} monitoring dashboards to view metrics for channels in your cluster.
+
+.Prerequisites
+
+* You have cluster administrator permissions, and access to the *Administrator* perspective in the {product-title} web console.
+* You have installed the {ServerlessOperatorName}, the Knative Eventing component, and an the `KnativeKafka` custom resource.
+* The {product-title} monitoring stack is enabled on your cluster. You can enable monitoring for {ServerlessProductName} during installation by checking the box to *Enable operator recommended cluster monitoring on this namespace* when installing the {ServerlessOperatorName}.
```

Comment:
@mikemckiernan I'll leave it this way, because your suggested formulation implies that the administrator needed to actively enable the stack. In reality, the stack is enabled by default, and this sentence is here just to make sure the administrator did not disable it.